dockerplugin.py

```diff
@@ -399,8 +400,73 @@ def __init__(self, docker_url=None):
         self.timeout = DockerPlugin.DEFAULT_DOCKER_TIMEOUT
         self.capture = False
         self.dimensions = None
+        self.excluded_labels = {}
+        self.excluded_images = []
+        self.excluded_names = []
         self.stats = {}
 
+    def is_excluded(self, container):
+        # exclude by image
+        image = container.get("Image", "")
+        for eximage in self.excluded_images:
+            try:
+                reg = re.compile(eximage)
```

Comment:
We should compile the regex patterns in `configure_callback` and iterate over the compiled regex objects.